From d9db885eaa2f9278f83ab2b977309f4a2552cd3b Mon Sep 17 00:00:00 2001
From: Guru9902 <gururaj.revankar@arm.com>
Date: Fri, 28 Nov 2025 08:25:42 +0000
Subject: [PATCH] add support compatible ID PNP0D15

Signed-off-by: Guru9902 <gururaj.revankar@arm.com>
---
 drivers/usb/host/xhci-plat.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/host/xhci-plat.c b/drivers/usb/host/xhci-plat.c
index 3d071b875..29233127f 100644
--- a/drivers/usb/host/xhci-plat.c
+++ b/drivers/usb/host/xhci-plat.c
@@ -557,6 +557,7 @@ EXPORT_SYMBOL_GPL(xhci_plat_pm_ops);
 static const struct acpi_device_id usb_xhci_acpi_match[] = {
 	/* XHCI-compliant USB Controller */
 	{ "PNP0D10", },
+	{ "PNP0D15", },
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, usb_xhci_acpi_match);
-- 
2.43.0

